10up/autoshare-for-twitter
# Added paths filter to GitHub actions

### Description of the Change
This PR adds a paths filter in GitHub action workflows to run it only on specific file extensions. For example, This PR doesn't contain any PHP, JS, or CSS file changes. So, the related workflows(E2E tests, PHPUnit, PHPCS, PHP Compatibility, and ESLint) will not be run against this PR. 


Closes #203 

### How to test the Change
See GitHub Actions check on this PR, E2E tests, PHPUnit, PHPCS, PHP Compatibility, and ESLint check should not run on this PR as we made changes in workflow files only.

### Changelog Entry
> Changed - Made GitHub Action workflow run only when required.

### Credits
Props @iamdharmesh

### Checklist:
